app/src/main/java/org/simple/clinic/appupdate/AppUpdateConfig.kt

```diff
@@ -0,0 +1,6 @@
+package org.simple.clinic.appupdate
+
+data class AppUpdateConfig(
+    val inAppUpdateEnabled: Boolean,
+    val numberOfVersionsToDiff: Long
```

Comment:
The number of versions to diff seems like it would be better off being remotely configurable. Are we planning to do this before releasing this feature?

Reply:
Yes. This is hard-coded for the time being.
Remote config will be implemented in another PR.